{{cookiecutter.pipeline_slug}}/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: nfcore-{{ cookiecutter.pipeline_name }}
```

Comment:
Maybe `nfcore-{{cookiecutter.pipeline_slug}}` instead (should not have whitespace, be lower case etc).